typescript/phoenix-sdk/src/client.ts

```diff
@@ -486,39 +489,51 @@ export class Client {
   }
 
   /**
-   * Get the price in ticks for a given price in quote units per base unit
-   * Example: For a tick size of 0.01 (quote units per base unit), a price in quote units per
-   * base unit of 1.23 would be 123 ticks
+   * Given a price in quote units per raw base unit, returns the price in ticks.
+   *
+   * Example: With a market tick size of 0.01, and a price of 1.23 quote units per raw base unit, the price in ticks is 123
+   *
    * @param price The price to convert
-   * @param marketAddress The `PublicKey` of the market account
+   * @param marketAddress The `PublicKey` of the market account, as a string
    */
   public floatPriceToTicks(price: number, marketAddress: string): number {
     const market = this.markets.get(marketAddress);
     if (!market) throw new Error("Market not found: " + marketAddress);
+
     return Math.round(
-      (price * 10 ** market.quoteToken.data.decimals) /
-        market.data.quoteLotsPerBaseUnitPerTick
+      (price *
```

Comment:
Check this math @jarry-xiao

Reply:
This looks fine to me, we can consider using `market.data.header.tickSizeInQuoteAtomsPerBaseUnit` in the denominator.